pkg/core/handler_test.go

```diff
@@ -40,6 +40,7 @@ var _ = Describe("Wave controller Suite", func() {
 	var m utils.Matcher
 
 	var deployment *appsv1.Deployment
+	var podControllerDeployment podController
```

Comment:
What do you think about reverting all of the changes in this file? If we have the `HandleDeployment` wrapper we shouldn't need these changes right? 😄